### ISSUE TITLE
low noise DC/DC converters for fast IO boards

### ISSUE THREAD

**Issue by gkasprow** [sinara-hw/meta#94]
We got several questions about noisy DIO modules.
The issue is the capacitance between the switching transformer winding and floating ground. For TMH1205S it can be as high as 100pF
That can be mitigated by increasing the capacitance between floating GND and chassis, but that would also lower the isolation barrier impedance.
I plan to implement improved LLC converters with 0.5pF coupling capacitance. That adds complexity and cost, and probably not everybody needs them, so the idea is to make them footprint-compatible with TMH1205S, and one could order standard or low-noise DIO modules

**Comment by hartytp** [comment]
> That adds complexity and cost, and probably not everybody needs them, so the idea is to make them footprint-compatible with TMH1205S, and one could order standard or low-noise DIO modules

What is the actual cost and complexity level we're talking about here?

Can I offer an alternative view on this? My guess (subject to an answer to the above) is that, at the volumes people are working at, the cost of support is greater than the BoM cost. Also, most users probably care more about having something which "just works" with good levels of performance, than they do about small optimizations to the BoM cost (the cost of debugging a noise issue in the lab is generally far greater than the cost of low noise parts).

So I'd suggest that either we think this is worth doing, in which case, once we've thoroughly prototyped it, we should make this the default (if not the only) supported option. If we think this is a niche thing then we shouldn't clutter up our designs with it. If we keep stacking up variants, the layouts are going to become a mess and the challenge of supporting users grows exponentially ("I can't help you debug this because I don't have the particular combination of hw variants you're using to hand").

**Comment by gkasprow** [comment]
I think you are right.
We will add a 7 EUR transformer, a 2 EUR controller, an optocoupler, and MOSFETs, making an extra ~20 EUR per converter.
So, make the default option or create a new product, like DIO-SMA-Low-Noise. In the case of a 16-channel DIOT module, that would add 80 EUR to the BOM.

**Comment by gkasprow** [comment]
Of course, TMH1205S is 7EUR/pc so the difference wouldn't be that high. Let's test it and make a default option, at least for one type of IO module.

**Comment by gkasprow** [comment]
I designed low noise DC/DC and sent for manufacturing

![Image](https://github.com/user-attachments/assets/d6eb5280-50ed-4527-a2bc-2d2c82591329)

**Comment by jordens** [comment]
@gkasprow Could you share the sources?

**Comment by gkasprow** [comment]
@jordens sure

[Ultra_LN_ISOL_SMPS.zip](https://github.com/user-attachments/files/18864050/Ultra_LN_ISOL_SMPS.zip)

**Comment by gkasprow** [comment]
It's modified TI design + feedback which the IC designer did not foresee :)

**Comment by gkasprow** [comment]
I already noticed some bugs, so wait until I make it run.

**Comment by MorganTL** [comment]
> I designed low noise DC/DC and sent for manufacturing

Btw what's the reasoning behind adding a common mode choke on the primary side?
